src/photon/photon.h

```diff
@@ -50,7 +44,7 @@ typedef struct scheduling_algorithm_state scheduling_algorithm_state;
  *  scheduler. */
 typedef struct {
   /** The script to use when starting a new worker. */
-  char *start_worker_command;
+  const char **start_worker_command;
```

Comment:
Why split the command into different strings? Is this so you can use `fork` instead of `popen`?